be/src/exec/vectorized/chunks_sorter_topn.cpp

```diff
@@ -531,6 +536,11 @@ Status ChunksSorterTopn::_hybrid_sort_first_time(RuntimeState* state, Permutatio
         ++index_of_merging;
     }
 
+    if (big_chunk->reach_capacity_limit()) {
+        LOG(WARNING) << "TopN sort exceed single chunk memory limit";
+        return Status::InternalError(fmt::format("TopN sort exceed single chunk memory limit"));
```

Comment:
This error can be misleading, and users may easily think that the memory limit is reached.

Reply:
fixed.